recipes/dask/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.9.0" %}
+
+package:
+  name: dask
+  version: "{{ version }}"
+
+source:
+  fn: dask-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/d/dask/dask-{{ version }}.tar.gz
+  md5: f111b837ae969a690cf23cd75add92e2
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - numpy
+    - toolz >=0.7.2
+    - pandas >=0.18.0
+    - partd >=0.3.3
+    - cloudpickle >=0.2.1
+
+  run:
+    - python
+    - numpy
+    - toolz >=0.7.2
+    - pandas >=0.18.0
+    - partd >=0.3.3
+    - cloudpickle >=0.2.1
+    - bokeh  # [not py26]
+
+test:
+  imports:
+    - dask
+    - dask.array
+    - dask.bag
+    - dask.bytes
+    - dask.dataframe
+    - dask.dataframe.tseries
+    - dask.delayed
+    - dask.diagnostics
+    - dask.store
+
+about:
+  home: http://github.com/dask/dask/
+  license: BSD 3-Clause
+  summary: 'Minimal task scheduling abstraction'
+
+extra:
+  recipe-maintainers:
+    - mrocklin
+    - jcrist
```

Comment:
Is it alright for both of you to be added as maintainers here, @mrocklin and @jcrist?